Gruntfile.js

```diff
@@ -9,7 +9,7 @@ module.exports = function (grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-contrib-uglify');
```

Comment:
My mistake - it makes development much faster. When you run grunt to compile on your local machine, with uglify it's about 20 seconds, without about 3 seconds. I'll put it back in.